2025-06-10 13:10:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-10 13:10:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-10 13:10:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-10 13:10:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-10 13:10:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-10 13:10:38 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-10 15:05:02 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-11 23:22:04 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SHOW COLUMNS [*]RANKING_ITEMS"; se esperaba "FROM"
Syntax error in SQL statement "SHOW COLUMNS [*]RANKING_ITEMS"; expected "FROM"; SQL statement:
SHOW COLUMNS RANKING_ITEMS [42001-214]
2025-06-11 23:22:06 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SHOW COLUMNS [*]RANKING_ITEMS ,"; se esperaba "FROM"
Syntax error in SQL statement "SHOW COLUMNS [*]RANKING_ITEMS ,"; expected "FROM"; SQL statement:
SHOW COLUMNS RANKING_ITEMS , [42001-214]
2025-06-11 23:22:06 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SHOW COLUMNS [*]RANKING_ITEMS"; se esperaba "FROM"
Syntax error in SQL statement "SHOW COLUMNS [*]RANKING_ITEMS"; expected "FROM"; SQL statement:
SHOW COLUMNS RANKING_ITEMS [42001-214]
2025-06-11 23:22:07 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SHOW COLUMNS [*]RANKING_ITEMS"; se esperaba "FROM"
Syntax error in SQL statement "SHOW COLUMNS [*]RANKING_ITEMS"; expected "FROM"; SQL statement:
SHOW COLUMNS RANKING_ITEMS [42001-214]
2025-06-11 23:22:39 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "SHOW COLUMNS FROM RANKING_ITEMS [*],"
Syntax error in SQL statement "SHOW COLUMNS FROM RANKING_ITEMS [*],"; SQL statement:
SHOW COLUMNS FROM RANKING_ITEMS , [42000-214]
2025-06-11 23:23:04 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "drop table  [*]from RANKING_ITEMS"; se esperaba "identifier"
Syntax error in SQL statement "drop table  [*]from RANKING_ITEMS"; expected "identifier"; SQL statement:
drop table  from RANKING_ITEMS [42001-214]
2025-06-11 23:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-11 23:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-11 23:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-11 23:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-11 23:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-11 23:24:17 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-26 17:44:36 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-26 17:52:59 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-06-27 00:08:11 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-07-02 12:16:13 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-07-02 12:43:42 jdbc[15]: null
org.h2.message.DbException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:190)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:132)
	at org.h2.Driver.connect(Driver.java:59)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:138)
	at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:359)
	at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:201)
	at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:470)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:733)
	at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:712)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1589)
Caused by: org.h2.jdbc.JdbcSQLNonTransientConnectionException: La conexión no fue cerrada por la aplicación y esta siendo limpiada (garbage collected)
The connection was not closed by the application and is garbage collected [90018-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	... 16 more
2025-07-02 13:24:58 jdbc[25]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
2025-07-07 12:20:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: El objeto ya está cerrado
The object is already closed [90007-214]
